bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -179,8 +179,6 @@ def call_plugin_route(self, base_plugin_path: str, method: str,
             query_params: The parameters of the query.
             request: The request object sent to the endpoint of the plugin.
         """
-        if query_params is None:
```

Comment:
Wouldn't this still be needed?

Reply:
No, because I changed it in the parameter `= []` assigns the default value if nothing was provided

Reply:
But what if you pass in `None`? Will it break?

Reply:
I think you're right that it would break if None is provided and lint complained so it's reverted now